src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball1 = new Lottery();
```

Comment:
1. Please create only one instance of the `Lottery` class and use it to create all the necessary balls. 
2. Also, please name it `Lottery lottery = new Lottery();`, because the ball is not a lottery.
3. Please, remove all other `Lottery` instances.